compiler/damlc/daml-doc/src/DA/Daml/Doc/Render/Markdown.hs

```diff
@@ -197,9 +197,15 @@ type2md env = f 0
     link :: Maybe Anchor -> Typename -> T.Text
     link Nothing n = escapeMd $ unTypename n
     link (Just anchor) n =
-        if renderAnchorAvailable env anchor
-            then T.concat ["[", escapeMd $ unTypename n, "](#", unAnchor anchor, ")"]
-            else escapeMd $ unTypename n
+        case lookupAnchor env anchor of
+            Nothing -> escapeMd $ unTypename n
+            Just anchorLoc -> T.concat
+                [ "["
```

Comment:
Maybe add `braces` and `parens` helpers?

Reply:
Or a `link` helper.